src/sentry/conf/server.py

```diff
@@ -1960,6 +1963,7 @@ def custom_parameter_sort(parameter: dict) -> tuple[str, int]:
             "project:read",
             "org:read",
             "member:read",
+            "member:invite",
```

Comment:
nit: ordering for all org roles
```suggestion
            "member:invite",
            "member:read",
```